src/main/java/com/google/cloud/hadoop/io/bigquery/hive/HiveBigQueryStorageHandler.java

```diff
@@ -63,127 +61,110 @@
 
     @Override
     public HiveMetaHook getMetaHook() {
-        return this;
-    }
-
-    @Override
-    public HiveAuthorizationProvider getAuthorizationProvider() throws HiveException{
```

Comment:
Why we remove this? Will it cause NPE?

Reply:
In Base Class (i.e DefaultStorageHandler) the method was already implemented with same code.